### DISCUSSION TITLE
Error in compilation & deployment

### DISCUSSION THREAD

**Discussion by MarkFS15** [smartcontractkit/full-blockchain-solidity-course-py#1898] [Q&A]
Hi there, the issue that i am currently facing is that my smart contract is unable to compile. It has been running and running non-stop.
![image 1](https://github.com/smartcontractkit/full-blockchain-solidity-course-py/assets/64194752/ed65e4cc-3c4a-4f73-a9a7-ba93e3b95104)
Due to that i am unable to deploy it. By the way, for the new remix ide version, how do we deploy it as i dont see a"deploy" button ?
![image](https://github.com/smartcontractkit/full-blockchain-solidity-course-py/assets/64194752/6ef8aa47-7542-4009-b397-c48cd82d2bd4)

**Comment by jvcByte** [comment]
hi @MarkFS15 a simple refresh of the browser can solve this.
